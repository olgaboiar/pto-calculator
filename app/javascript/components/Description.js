import React from 'react'

class Description extends React.Component {
  render () {
    return (
      <div
        data-testid="description"
        >The only tool that brings your vacation closer!</div>
    )
  }
}

export default Description
