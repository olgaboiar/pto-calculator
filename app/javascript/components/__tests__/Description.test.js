import { shallow, render } from "enzyme";
import * as React from "react";
import Description from "../Description"

describe("Description.js", () => {
  it("renders correctly", () => {  
    const wrapper = shallow(
      <Description />,
    );
  
    expect(wrapper).toMatchSnapshot();
  });

  it("shows brief description", () => {  
    const wrapper = render(
      <Description />,
    );
  
    expect(wrapper.text()).toContain("The only tool that brings your vacation closer!");
  });
})
