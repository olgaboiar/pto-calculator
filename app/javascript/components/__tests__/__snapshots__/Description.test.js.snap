// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description.js renders correctly 1`] = `ShallowWrapper {}`;
